lib/vagrant-openshift/constants.rb

```diff
@@ -52,7 +52,10 @@ def self.repos_for_name(reponame)
           'origin' => openshift_repos,
           nil => openshift_repos,
           'origin-console' => console_repos,
-          'origin-aggregated-logging' => aggregated_logging_repos
+          'origin-aggregated-logging' => aggregated_logging_repos,
+          'origin-metrics' => {
```

Comment:
This isn't a valid hash in the `{...}`; I would replace it with a method that returns a hash, to match the pattern of the rest of the file.